cli/tests/cli-reference@.md.snap

```diff
@@ -364,17 +375,19 @@ For information about bookmarks, see https://jj-vcs.github.io/jj/latest/bookmark
 
    Can be combined with `--tracked` or `--conflicted` to filter the bookmarks shown (can be repeated.)
 
-   By default, the specified remote name matches exactly. Use `glob:` prefix to select remotes by wildcard pattern. For details, see https://jj-vcs.github.io/jj/latest/revsets/#string-patterns.
+   By default, the specified remote name matches exactly. Use `glob:` prefix to select remotes by [wildcard pattern].
+
+   [wildcard pattern]: https://jj-vcs.github.io/jj/latest/revsets/#string-patterns
 * `-t`, `--tracked` — Show remote tracked bookmarks only. Omits local Git-tracking bookmarks by default
 * `-c`, `--conflicted` — Show conflicted bookmarks only
 * `-r`, `--revisions <REVSETS>` — Show bookmarks whose local targets are in the given revisions
 
    Note that `-r deleted_bookmark` will not work since `deleted_bookmark` wouldn't have a local target.
 * `-T`, `--template <TEMPLATE>` — Render each bookmark using the given template
 
-   All 0-argument methods of the `RefName` type are available as keywords.
+   All 0-argument methods of the [`RefName` type] are available as keywords in the [template expression].
 
-   For the syntax, see https://jj-vcs.github.io/jj/latest/templates/
+   [template expression]: https://jj-vcs.github.io/jj/latest/templates/ [`RefName` type]: https://jj-vcs.github.io/jj/latest/templates/#refname-type
```

Comment:
I just keep coming up with trouble. 😞 

This doesn't render correctly in MkDocs. We either need to fix our Markdows generator or insert a paragraph, I guess?

Reply:
fixed